client/index.js

```diff
@@ -9,14 +9,15 @@ import * as asset from '../lib/asset'
 import * as envConfig from '../lib/runtime-config'
 import ErrorBoundary from './error-boundary'
 import Loadable from 'react-loadable'
+import PromisePolyfill from 'promise-polyfill'
```

Comment:
core-js has the promise polyfill, and it's still being installed afaik, so can we use that one?